config/app.php

```diff
@@ -243,4 +243,29 @@
         'verification_sid' => env('TWILIO_VERIFICATION_SID'),
     ],
 
+    /*
+    |--------------------------------------------------------------------------
+    | Debug blacklist
+    |--------------------------------------------------------------------------
+    |
+    | Hiding environment variables on debug pages. For more information, visit
+    | https://laravel.com/docs/7.x/configuration#hiding-environment-variables-from-debug
+    |
+    */
+
+    'debug_blacklist' => [
```

Comment:
@fefi95 Based on the documentation added as comment, should it be `debug_hide` ?

Reply:
Good catch! The `debug_blacklist` was the name on version 5.8 but in 7 is `debug_hide`. I changed it and it works the same (`debug_blacklist` was compatible).